app/src/main/java/me/hufman/androidautoidrive/carapp/music/views/BrowsePageView.kt

```diff
@@ -373,23 +399,30 @@ class BrowsePageView(val state: RHMIState, val musicImageIDs: MusicImageIDs, val
 				if (item == SEARCHRESULT_EMPTY || item == SEARCHRESULT_SEARCHING) {
 					// invalid selection, don't change states
 					inputComponent.getSuggestAction()?.asHMIAction()?.getTargetModel()?.asRaIntModel()?.value = 0
+
 					throw RHMIActionAbort()
 				} else if (item == BrowseView.SEARCHRESULT_PLAY_FROM_SEARCH) {
 					browseController.playFromSearch(this.input)
 					browseController.onListSelection(item, inputComponent.getSuggestAction()?.asHMIAction())
 				} else {
```

Comment:
Can a new SEARCHRESULT be added to the list to say something like "View Full Results" which would do the same `showSearchResults` as the Ok button?

Reply:
Added a new "View Full Results" entry to the suggestion model along with a "..." entry at the end of the truncated suggestion list (due to the suggestionsModel only accurately showing the first 32 items) if the list is indeed truncated. Both of these new entries when clicked will open the SearchResultsView